### PR TITLE
Basic views

### PR DESCRIPTION
**Pull request by jayrbolton**
This adds a couple very basic, generic views. You can run these on any collection. I'm using these in API tests.

Other generic view ideas:
- Traverse from vert to vert via one level of edges, two levels of edges, etc.
- List all documents in a collection with an equality filter on an attribute

Idea to make views more flexible: allow flags that add or remove sections of the query (eg a filter line)

### CODE REVIEW

File: views/list_all_documents_in_collection.aql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Return *all* full documents in a collection
+// Args:
+//   collection - name of collection to count docs
+
+for v in @@collection
+  return v
```
Comment (JamesJeffryes) [inline views/list_all_documents_in_collection.aql]: We might want to put a limit on this so someone does not tie up Arango by running this on ws_object_versions. We also need to give another think on access control.

Comment (jayrbolton) [reply views/list_all_documents_in_collection.aql]: The API will limit things. Query results are limited to 100 and more result pages can be fetched through a cursor. RAM usage for a single query is also capped. I think this should be fine in prod actually.

Comment (jayrbolton) [reply views/list_all_documents_in_collection.aql]: But regarding access control -- yes. We'll have to update this for that.

Comment (JamesJeffryes) [reply views/list_all_documents_in_collection.aql]: so `Return *all* full documents in a collection` is a LIE! Good to know about the caps on queries though.

Comment (jayrbolton) [reply views/list_all_documents_in_collection.aql]: Not a lie if you follow the magic cursor!
Say you had 200 docs in a collection:
- POST /query {name: list_all_documents_in_collection} -> 100 results, cursor 123
- GET /cursor {id: 123} -> 100 next results, no cursor

Comment (jayrbolton) [reply views/list_all_documents_in_collection.aql]: I might change the above API to make things a little less confusing:

* GET /query_results {view: lsit_all_documetns_blah} -> cursor 123
* GET /query_resutls {cursor_id: 123}

**Pull request merged**